include/seastar/http/request.hh

```diff
@@ -277,7 +277,7 @@ struct request {
     static request make(httpd::operation_type type, sstring host, sstring path);
 
 private:
-    void add_param(const std::string_view& param);
+    void add_query_param(const std::string_view& param);
```

Comment:
Can do `s/const std::string_view&/std::string_view/` in the same change as well

Reply:
This suggestion is still not applied

Reply:
Sorry, I first thought that you meant to ask for me to squash the first 2 commits, the one introducing the `path_decode` function and the one that did the `s/const std::string_view&/std::string_view/` for `url.hh` and `url.cc`.

But I think you meant that I should do the same `s/const std::string_view&/std::string_view/` in `request.cc` and `request.hh` as well. I've done that now (see [force-push](https://github.com/scylladb/seastar/compare/79c509e748f20e52726dbb0eb8ef83b1621c24a9..d03443606044ecb0978a290461aaa9766f0cd01b)).